src/Enchantments.h

```diff
@@ -149,7 +159,19 @@ class cEnchantments
 
 	/** Currently stored enchantments */
 	cMap m_Enchantments;
-} ;  // tolua_export
+
+public:
+	/** Make this class iterable */
+	cMap::iterator begin() { return m_Enchantments.begin(); }
+	cMap::iterator end()   { return m_Enchantments.end(); }
+
+private:
+	/** A vector of all of the mutually-exclusive sets of enchantments */
+	std::vector<std::set<int> > m_IncompatibleEnchantments;
```

Comment:
Why exactly does each `cEnchantments` object have a copy of this?

Personally, I'd make a static function that would return the data, and use the C++ singleton approximation (`static const` object inside the function) for a single initialization point.